includes/admin/tools.php

```diff
@@ -1350,6 +1350,7 @@ function edd_tools_sysinfo_get() {
 	$return .= 'Language:                 ' . ( ! empty( $locale ) ? $locale : 'en_US' ) . "\n";
 	$return .= 'Permalink Structure:      ' . ( get_option( 'permalink_structure' ) ? get_option( 'permalink_structure' ) : 'Default' ) . "\n";
 	$return .= 'Active Theme:             ' . $theme . "\n";
+	$return .= 'Timezone:                 ' . wp_timezone_string() . "\n";
```

Comment:
Since we're really curious about what timezone EDD is using for charts/graphs, could we include the result from this function as well:
`edd_get_timezone_abbr()`